client/src/component/Review.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+class Review extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    const { review } = this.props;
+    const {
+      profile_pic_url, username, time_made, message 
+    } = review;
+    return (
+      <div>
+        <div>
+          <div>
+            <div><img src={profile_pic_url} alt="" /></div>
+            <div>{username}</div>
+            <div>{time_made}</div>
+          </div>
+        </div>
+        <h5>
```

Comment:
Because the message is not a header maybe decide on using a different tag to wrap the message and do the styling in CSS